cmd/guide.go

```diff
@@ -19,253 +23,237 @@ func insertGuideTasks(db *sql.DB) error {
 	var entries = []entry{
 		{
 			"guide: welcome to omm",
-			`Hi there 👋 Thanks for trying out omm.
+			fmt.Sprintf(`Hi there 👋 Thanks for trying out **omm**.
```

Comment:
Maybe you could use go:embed here.

You would create separate Markdown files, then you won't have to play with sprintf to add ` in each text.

Also,it would be easier to read and maintain I think

Reply:
Great idea.

Reply:
Done. Much cleaner this way.